### PR TITLE
Chinese translation and placeholder.

### PR DESCRIPTION
**Pull request by squirrelsc**

### CODE REVIEW

File: README_zh_CN.md [hunk 1]
```diff
@@ -1,9 +1,8 @@
 # AI 开放平台（OpenPAI） ![alt text](./pailogo.jpg "OpenPAI")
 
-[![生成状态](https://travis-ci.org/microsoft/pai.svg?branch=master)](https://travis-ci.org/microsoft/pai)
-[![代码覆盖状态](https://coveralls.io/repos/github/microsoft/pai/badge.svg?branch=master)](https://coveralls.io/github/microsoft/pai?branch=master)
-[![进入 https://gitter.im/Microsoft/pai 聊天室提问](https://badges.gitter.im/Microsoft/pai.svg)](https://gitter.im/Microsoft/pai?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
-[![版本](https://img.shields.io/github/release/Microsoft/pai.svg)](https://github.com/Microsoft/pai/releases/latest)
+[![生成状态](https://travis-ci.org/microsoft/pai.svg?branch=master)](https://travis-ci.org/microsoft/pai) [![代码覆盖状态](https://coveralls.io/repos/github/microsoft/pai/badge.svg?branch=master)](https://coveralls.io/github/microsoft/pai?branch=master) [![进入 https://gitter.im/Microsoft/pai 聊天室提问](https://badges.gitter.im/Microsoft/pai.svg)](https://gitter.im/Microsoft/pai?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge) [![版本](https://img.shields.io/github/release/Microsoft/pai.svg)](https://github.com/Microsoft/pai/releases/latest)
+
```
Comment (abuccts) [inline README_zh_CN.md]: why write one line for multiple badges?
e.g. https://github.com/microsoft/vscode/blame/master/README.md#L4-L7

File: README_zh_CN_zh_CN.md [hunk 2]
```diff
@@ -0,0 +1,171 @@
+# AI 开放平台（OpenPAI） ![alt text](./pailogo.jpg "OpenPAI")
```
Comment (abuccts) [inline README_zh_CN_zh_CN.md]: duplicate file and wrong filename

File: contrib/marketplace/README_zh_CN.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
+# Marketplace Plugin
```
Comment (abuccts) [inline contrib/marketplace/README_zh_CN.md]: could you do not add English "zh_CN.md" files until Chinese version is available?

File: examples/Dockerfiles/autobuild_docker/README_zh_CN_zh_CN.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
+# How to maintain docker image by using dockerfile
```
Comment (abuccts) [inline examples/Dockerfiles/autobuild_docker/README_zh_CN_zh_CN.md]: wrong filenames

File: README_zh_CN.md (hunk 1 above)
Comment (squirrelsc) [reply README_zh_CN.md]: It's translated by crowdin.com automatically. If there is no UI impact, it's ok.

File: contrib/marketplace/README_zh_CN.md (hunk 3 above)
Comment (squirrelsc) [reply contrib/marketplace/README_zh_CN.md]: It's using a same rule to copy files. I will try to remove them, if they are out of docs folder. For the files in docs folder, they will be kept to keep links works cross file.

File: README_zh_CN.md (hunk 1 above)
Comment (abuccts) [reply README_zh_CN.md]: The source is hard for people to read or edit if you write in one line.

File: contrib/storage_plugin/README_zh_CN.MD [hunk 5]
```diff
@@ -3,27 +3,28 @@
 A tool to manage external storage in PAI.
```
Comment (abuccts) [inline contrib/storage_plugin/README_zh_CN.MD]: Use `.md` in filename instead of `.MD`?

File: docs/zh_CN/rest-server/API.md [hunk 6]
```diff
@@ -716,12 +716,103 @@ Status: 404
     <br />*Response if a server error occured*
     
 
+Status: 500
+
+{ "code": "UnknownError", "message": "*Upstream error messages*" }
+
+    <br /><br />## API v2
```
Comment (abuccts) [inline docs/zh_CN/rest-server/API.md]: there's format issue, do not need `<br>` here

File: README_zh_CN.md (hunk 1 above)
Comment (squirrelsc) [reply README_zh_CN.md]: It includes some icons, and show as online in both English and Chinese. Only difference is in source, not in rendered result.

File: README_zh_CN_zh_CN.md (hunk 2 above)
Comment (squirrelsc) [reply README_zh_CN_zh_CN.md]: good catch. removed.

File: examples/Dockerfiles/autobuild_docker/README_zh_CN_zh_CN.md (hunk 4 above)
Comment (squirrelsc) [reply examples/Dockerfiles/autobuild_docker/README_zh_CN_zh_CN.md]: removed.

File: contrib/storage_plugin/README_zh_CN.MD (hunk 5 above)
Comment (squirrelsc) [reply contrib/storage_plugin/README_zh_CN.MD]: the name is generated by tool, and exactly same as English file name. to get consistence, the English file name should be changed firstly.

File: docs/zh_CN/rest-server/API.md (hunk 6 above)
Comment (squirrelsc) [reply docs/zh_CN/rest-server/API.md]: It's due to error format in English, I will fix it.

File: README_zh_CN.md [hunk 7]
```diff
@@ -1,9 +1,8 @@
...
+[English](README.md)
```
Comment (abuccts) [inline README_zh_CN.md]: duplicate line

Comment (squirrelsc) [reply README_zh_CN.md]: fixed

**Pull request merged**